src/tools/std20/views/test/single.test.cpp

```diff
@@ -0,0 +1,40 @@
+// include Catch2
+#include <catch2/catch_test_macros.hpp>
+
+// what we are testing
+#include "tools/std20/views.hpp"
+
+// other includes
+
+// convenience typedefs
+//using namespace njoy::tools;
+namespace std20 = nano;
+
+SCENARIO( "single_view" ) {
+
+  auto chunk = std20::views::single( 3 );
+  using Range = decltype(chunk);
+  using Iterator = nano::iterator_t< Range >;
+
+  THEN( "the take_view satisfies the required concepts" ) {
```

Comment:
"take_view" -> "single" 

Likewise for other THENs in scenario

Reply:
same